internal/drivers/basic/fixtures/account_management.yaml

```diff
@@ -0,0 +1,38 @@
+accounts:
+- name: abcde
+  auth_tenant_id: "1245"
+  gc_policies:
+  - match_repository: .*/database
+    except_repository: archive/.*
+    time_constraint:
+      "on": pushed_at
+      newer_than: 21600000000000
```

Comment:
Since this file is supposed to be human-writable, this should use a more accessible format. We have a precedent for this in https://pkg.go.dev/github.com/sapcc/limes/internal/util#MarshalableTimeDuration, so this would make sense for go-bits (maybe a new subpackage "timeext").